metecho/api/jobs.py

```diff
@@ -566,9 +566,12 @@ def get_social_image(*, project):
         logger.error(tb)
         raise
     else:
-        project.refresh_from_db()
-        project.repo_image_url = og_image
-        project.finalize_get_social_image()
+        # Save the image only if it was manually set by the repository owner. GitHub
+        # seems to store only manually-set images under this domain:
+        if og_image.startswith("https://repository-images.githubusercontent.com/"):
+            project.refresh_from_db()
+            project.repo_image_url = og_image
+            project.finalize_get_social_image()
```

Comment:
- If deployed to an environment without first clearing the repo images the old images will persist because this doesn't remove "wrong" URLs, just skips saving. We can modify the logic to set project.repo_image_url = "" if we want to be strict.
- This "check the domain" trick is undocumented and Might Break™ without warning. If you're curious the auto-generated images seem to always come from https://opengraph.githubassets.com.

Reply:
It's okay to leave existing autogenerated images; it's easy enough to fix when we see them.